client/js/models/archive.js

```diff
@@ -72,9 +78,14 @@ module.exports = {
   ],
   effects: {
     new: function (data, state, send, done) {
-      const archive = drive.createArchive(null, {live: true, sparse: true})
+      drive = getDrive()
+      const archive = drive.createArchive(null, {
+        live: true,
+        sparse: true,
+        file: ram
```

Comment:
yeah, should be `file: () => ram()` otherwise it will simulate a file with the file path as its content, as `file: path => ram(path)` will be interpreted as `ram(Buffer(path))`